sdjournal/journal.go

```diff
@@ -67,10 +230,21 @@ const (
 	IndefiniteWait time.Duration = 1<<63 - 1
 )
 
+var libsystemdNames = []string{
+	// systemd < 209
+	"libsystemd-journal.so",
```

Comment:
Shouldn't this be reversed? First the one with the `.0` suffix then the one without?